src/NuGet.Clients/NuGet.Console/OutputConsole/OutputConsole.cs

```diff
@@ -45,21 +44,26 @@ internal sealed class OutputConsole : IConsole, IConsoleDispatcher
 
             _outputWindowPane = new Lazy<IVsOutputWindowPane>(() =>
             {
-                // create the Package Manager pane within the Output window
-                int hr = _vsOutputWindow.CreatePane(
-                    ref GuidList.guidNuGetOutputWindowPaneGuid,
-                    Resources.OutputConsolePaneName,
-                    fInitVisible: 1,
-                    fClearWithSolution: 0);
-                ErrorHandler.ThrowOnFailure(hr);
-
-                IVsOutputWindowPane pane;
-                hr = _vsOutputWindow.GetPane(
-                    ref GuidList.guidNuGetOutputWindowPaneGuid,
-                    out pane);
-                ErrorHandler.ThrowOnFailure(hr);
-
-                return pane;
+                return NuGetUIThreadHelper.JoinableTaskFactory.Run(async () =>
+                {
+                    await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+
```

Comment:
Thread affinity within a `Lazy<T>` is prone to deadlocks. Use `AsyncLazy<T>` instead.
We have an analyzer on the backlog to help catch this anti-pattern: [https://github.com/Microsoft/vs-threading/issues/253](https://github.com/Microsoft/vs-threading/issues/253)